installer/scripts/service_control

```diff
@@ -69,29 +145,53 @@ verify_privileges()
     fi
 }
 
+this_omsagent_running() {
+    if [ -f $PIDFILE ]; then
+        if ps -p `cat $PIDFILE` | grep -q omsagent
```

Comment:
Can you make this check more explicit? I would rather have you execute the ps/grep command and then check $? like `is_omsagent_running()` did previously. As it is, it looks a little unclear.